pages/content-ui/vite.config.mts

```diff
@@ -1,24 +1,38 @@
 import { resolve } from 'node:path';
-import { makeEntryPointPlugin } from '@extension/hmr';
 import { isDev, withPageConfig } from '@extension/vite-config';
+import { processCssPlugin } from '@extension/hmr';
+import { getEntryPoints } from './script/entryPoints';
 
 const rootDir = resolve(__dirname);
 const srcDir = resolve(rootDir, 'src');
 
+/**
+ * Here is the place to add new entry points, u can straight away add new folders here to include
+ * without creating folders in src dir manually.
+ * @param {string[]} options.include - Folders to include !!MODIFY ME!!
+ * @param {string[]} options.exclude - Folders to exclude !!MODIFY ME!!
+ * @param {boolean} options.createRecursively - Create folders recursively
+ * @returns {Record<string, string>}
+ */
+const entryPoints = getEntryPoints({ 
+  include: ['page1', 'page2'], createRecursively: true 
+});
```

Comment:
It seems more like `makeEntryponits` than `getEntryPoints`, and shouldn't it be possible to run it as a separate script? I'm curious why it's in the vite config together.

Reply:
cause getentrypoints/makeentrypoints is resolving to the array of folder names to be used in vite.config entries. that's why its here. if not vite config don't know which folder to process. And yes this is more of convenience feature and people can add to manifest.json later